modules/swagger-parser-v3/src/test/resources/oas3.fetched/components/parameters/Exclude.yaml

```diff
@@ -0,0 +1,12 @@
+description: |-
```

Comment:
In a pinch, I added the impacted spec as a test resource, but I need to trim it down to just the parts necessary for the new test.